catalyst_voices/apps/voices/lib/pages/registration/create_account_progress/account_create_progress_panel.dart

```diff
@@ -0,0 +1,148 @@
+import 'package:catalyst_voices/pages/registration/widgets/next_step.dart';
+import 'package:catalyst_voices/pages/registration/widgets/registration_progress_stepper.dart';
+import 'package:catalyst_voices/widgets/buttons/voices_filled_button.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+
+class AccountCreateProgressPanel extends StatelessWidget {
+  final List<AccountCreateStepType> completedSteps;
+
+  const AccountCreateProgressPanel({
+    super.key,
+    required this.completedSteps,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final lastCompletedStep = completedSteps.lastOrNull;
+
+    final title = lastCompletedStep?._title(context);
+
+    final nextStep = lastCompletedStep?.next;
+    final nextStepText = lastCompletedStep?._nextStepText(context);
+
+    return Column(
+      key: const Key('AccountCreateProgressPanel'),
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        const SizedBox(height: 24),
+        Expanded(
+          child: SingleChildScrollView(
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
+                if (title != null) ...[
+                  _TitleText(title),
+                  const SizedBox(height: 24),
+                ],
+                RegistrationProgressStepper(
+                  completed: completedSteps.toSet(),
+                  current: [...AccountCreateStepType.values]
```

Comment:
Is recreating a list here needed?
```suggestion
                  current: AccountCreateStepType.values
```

Reply:
No longer needed. I was deleting items before and forgot to change it back later